### PR TITLE
fixed poolchairs

### LINKED ISSUE #11761: Invisible chairs
### Labels

[BUG][MINOR]

### Description

Pool folding chairs are invisible while folded.

### Steps to reproduce

Click folding chair
Get the message they are folded
Sprite is invisible, still available through right-click

### Expected Behavior

Visible chair

### Additional Information & Screenshots

Reported by: GreenJelly
On server: Goonstation 3 Roleplay: Morty
Round log date: 2022-11-02-03-20
Reported on: 2022-11-02 04:38:36

### PR DESCRIPTION
**Pull request by TheTyrant25**
[BUG] [MINOR]
## About the PR 
Fixes #11761 by making pool chairs a subtype of chair that can't be folded or rotated.

### CODE REVIEW

**Comment by TheTyrant25** [comment]
Pool chairs were literally just varedited regular folding chairs `/obj/stool/chair`, so this fixes that. Will make further commits to replace them in every map. Question is though, is them being foldable intended? Because pool chairs have literally one sprite and that is the `SOUTH` sprite and i don't wanna do spritework lol

File: code/obj/stool.dm [hunk 1]
```diff
@@ -1523,3 +1523,14 @@
...
+/* ========================================================= */
+/* ---------------------- Pool Chairs ---------------------- */
+/* ========================================================= */
+/obj/stool/chair/pool
+	name = "pool chair"
+	desc = "This chair is perfect for lounging at the side of a pool."
+	icon_state = "chair_pool"
+	rotatable = 0
+	foldable = 0
+	climbable = 0
```
[the remark this answers is not in the document]
Comment (TheTyrant25) [reply code/obj/stool.dm:1536]: i- huh. why did i do that. will remove that shortly

**Comment by TheTyrant25** [comment]
uhhh @amylizzle it dont work anymore

**Review by amylizzle — APPROVED** [review]
LGTM. Delete the changelog though, not much point for bugfixes. Probably also ditch the input-wanted tag too, since there's not a lot to input on.

**Comment by TheTyrant25** [comment]
> LGTM. Delete the changelog though, not much point for bugfixes. Probably also ditch the input-wanted tag too, since there's not a lot to input on.

gotcha

**Review by pali6 — APPROVED** [review]

**Pull request merged**